### PR TITLE
global: travis

### PR DESCRIPTION
**Pull request by crepererum**
* Implements basic test workflow for travis.

* Adds `.gitignore` also to root folder.

Signed-off-by: Marco Neumann <marco@crepererum.net>

### CODE REVIEW

**Comment by crepererum** [comment]
Waiting for the upstream fix (Invenio 2.0.3) to release on PyPi.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Cookiecutter - Invenio Module Template
+# Copyright (C) 2015 CERN
+#
+# Invenio is free software; you can redistribute it and/or
```
Comment (jirikuncar) [inline .travis.yml]: `s/Invenio/Cookiecutter - Invenio Module Template/`?

File: {{ cookiecutter.project_shortname }}/setup.py [hunk 2]
```diff
@@ -34,16 +34,16 @@
 history = open('CHANGES.rst').read()
 
 requirements = [
-    'Flask',
-    'six',
-    'Invenio',
+    'Flask>=0.10.1',
+    'six>=1.7.2',
+    'Invenio>=2.0.0',
```
Comment (jirikuncar) [inline {{ cookiecutter.project_shortname }}/setup.py]: I would even say 2.0.3 ;)

**Comment by crepererum** [comment]
The PEP8 failure should be fixed by #7, so I'm waiting for this one to get merged.

**Comment by lnielsen** [comment]
Can you rebase and repush. Just merged #7 and #9

**Comment by crepererum** [comment]
done

**Pull request merged**